src/Controller/IndexController.php

```diff
@@ -135,6 +134,9 @@ public function ajaxAction() {
 		$output->setFormatter(new HtmlOutputFormatter($formatter));
 
 		$application->setAutoExit(false);
+
+		$endpoint = preg_replace('/(updater\/|updater\/index.php)$/', '', $this->request->server('REQUEST_URI'));
```

Comment:
I'm honestly bad with regex :see_no_evil: 

Will this work for setups where index.php is not part of the url?

THX